hmmlearn/base.py

```diff
@@ -504,6 +504,11 @@ def _init(self, X, lengths):
     def _check(self):
         """Validates model parameters prior to fitting.
 
+        Note: Only startprob_ and transmat_ are covered by this method, as 
```

Comment:
I understand your confusing here, but I think this comment could be inferred from the fact that `_BaseHMM` is a base class and it only validates the parameters it defines.